vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2225,6 +2258,53 @@ private void launchFileSelectionIntent() {
         startActivityForResult(fileIntent, REQUEST_FILES_REQUEST_CODE);
     }
 
+    private void startStickerPickerActivity() {
+        // Search for the sticker picker widget in the user account
+        Map<String, Object> userWidgets = mSession.getUserWidgets();
+
+        String stickerWidgetUrl = null;
+        String stickerWidgetId = null;
+
+        for (Object o : userWidgets.values()) {
+            if (o instanceof Map) {
+                Object content = ((Map) o).get("content");
+                if (content != null && content instanceof Map) {
+                    Object type = ((Map) content).get("type");
+                    if (type != null && type instanceof String && type.equals(StickerPickerActivity.WIDGET_NAME)) {
+                        stickerWidgetUrl = (String) ((Map) content).get("url");
+                        stickerWidgetId = (String) ((Map) o).get("id");
+                        break;
+                    }
+                }
+            }
+        }
+
+        if (TextUtils.isEmpty(stickerWidgetUrl)) {
+            // The Sticker picker widget is not installed yet. Propose the user to install it
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
+
+            // Use the builder context
+            View v = LayoutInflater.from(builder.getContext()).inflate(R.layout.no_sticker_pack_dialog, null);
+
+            builder
+                    .setView(v)
+                    .setPositiveButton(R.string.yes, new DialogInterface.OnClickListener() {
+
+                        @Override
+                        public void onClick(DialogInterface dialog, int which) {
+                            // Open integration manager
+                            openIntegrationManagerActivity();
```

Comment:
We should open the integration manager on the sticker packs settings screen, no?

Reply:
Yes, It should be great, but we do not know the `widgetId`.